pallets/transaction-multi-payment/src/lib.rs

```diff
@@ -248,6 +250,26 @@ decl_module! {
 
 			Ok(())
 		}
+
+		fn on_runtime_upgrade() -> frame_support::weights::Weight {
+			mod previous {
+				pub struct Module<T>(sp_std::marker::PhantomData<T>);
+				frame_support::decl_storage! {
+					trait Store for Module<T: super::Config> as TransactionPayment {
+						pub AcceptedCurrencies get(fn currencies) config(): super::Vec<super::AssetId>;
+					}
+				}
+			}
+			let version = <Self as GetPalletVersion>::storage_version();
+
+			if version == None || version == Some(PalletVersion::new(3, 0, 0)) {
+				previous::AcceptedCurrencies::kill();
+				AcceptedCurrencies::put(OrderedSet::<AssetId>::new());
+				T::BlockWeights::get().max_block
```

Comment:
Is the PalletVersion value updated automatically to 3.0.0, resp the current crate version ?

Reply:
it should be